src/main/java/com/redhat/red/build/koji/model/xmlrpc/messages/GetTaskRequest.java

```diff
@@ -28,14 +28,29 @@
     @DataIndex( 0 )
     private int taskId;
 
+    @DataIndex( 1 )
+    private boolean request;
+
```

Comment:
I'm not familiar with xml rpc call. Does this request param has a default value? If send with no value, its default is false/true? Seems this change will send false as default(if not specified), will it have any influence on other calls?